element_calcium_imaging/imaging.py

```diff
@@ -152,8 +151,23 @@ def make(self, key):
             else:
                 raise NotImplementedError('Unknown method: {}'.format(method))
         elif task_mode == 'trigger':
-            raise NotImplementedError(f'Automatic triggering of {method} analysis'
-                                      f' is not yet supported')
+            method = (ProcessingTask * ProcessingParamSet * ProcessingMethod * scan.Scan & key).fetch1('processing_method') 
+            if method == 'suite2p':
+                import suite2p
+
+                suite_ops = (ProcessingTask * ProcessingParamSet & key).fetch1('params')
+                suite_ops['save_path0'] = (ProcessingTask & key).fetch1('processing_output_dir')
+
+                db = {}                
+                db['data_path'] = (ProcessingTask * scan.Scan * scan.ScanInfo * scan.ScanInfo.ScanFile & key).fetch('file_path')[0]
+                db['data_path'] = [find_full_path(get_imaging_root_data_dir(), db['data_path']).as_posix()]  # Suite2p requires data_path to be a list.
+                
+                _ = suite2p.run_s2p(ops=suite_ops, db=db)  # Run suite2p
```

Comment:
You don't really need to assign the return value to `_` (or anything) if you are not using it.

Reply:
It's just a dummy variable keeps the screen clean, happy to not use it

Reply:
It suppresses the output when this is the last line in a Jupyter cell. But this is not a jupyter cell.

Reply:
I'm removing the dummy variable `_`. It won't appear in the next PR.